piston/validate_jsonp.py

```diff
@@ -140,16 +140,16 @@ def test():
       >>> is_valid_javascript_identifier('$210')
       True
 
-      >>> is_valid_javascript_identifier(u'Stra\u00dfe')
+      >>> is_valid_javascript_identifier(u'Stra\\u00dfe')
```

Comment:
Didn't update the comments correctly here, re: `u'string'` unicode string classifier.